integrations/facebook-pixel/test/index.test.js

```diff
@@ -1344,6 +1343,33 @@ describe('Facebook Pixel', function() {
         );
         assertEventId(window.fbq);
       });
+
+      it('should send facebook pixel properties for product viewed', function() {
+        analytics.track('Product Viewed', {
+          product_id: '507f1f77bcf86cd799439011',
+          contents: [
```

Comment:
A `contents` array shouldn't appear in a Segment track event, but the logic should maps products from a Segment event to a Facebook `contents` array. Example: https://github.com/webonise/analytics.js-integrations/blob/master/integrations/facebook-pixel/test/index.test.js#L2047-L2050